app/model/Export/templates/educationReport.latte

```diff
@@ -22,13 +22,19 @@ $chits - paragony akce
     <h1>{$education->getDisplayName()}</h1>
 
     <h3>Základní údaje</h3>
-    {var $functionsWithName = [
-    'Vedoucí tábora' => $functions->leader,
-    'Zástupce vedoucího tábora' => $functions->assistant,
+
+    <b>Vedoucí akce: </b> {if $functions->leader !== NULL}{$functions->leader->name} {if $functions->leader->email !== NULL}(e-mail: {$functions->leader->email}){/if}{/if}<br>
```

Comment:
Proč se má zobrazovat email v hospodareni vzdělávačky? Hospodář předpokládám zná email vedoucího a zástupce

Reply:
Protože jsem to zkopíroval ze šablony pro závěrečku tábora :D Podle mě tam klidně být nemusí (ale to u táborů asi taky ne)...

Reply:
začal bych tím, že to tady odeberem. Pak můžeme hledat proč a kdo ho tam kdy přidal u táborů